### PR TITLE
Adding license in readme

### LINKED ISSUE #142: Clarify in the documentation the BSD nature of the license.
For the introductory readme, I mean.

### PR DESCRIPTION
**Pull request by stefanoborini**
Fixes #142

### CODE REVIEW

**Comment by kitchoi** [comment]
Is this included in the sphinx doc?  I just tried `make docs` and I could not find a page for the license.

**Comment by stefanoborini** [comment]
@kitchoi then why aren't the tests failing?

**Comment by kitchoi** [comment]
@stefanoborini Tests did not fail because the doc is built successfully.  I meant there isn't a page that mentions license on the doc.  Isn't it what #142 meant?
screenshot from 2016-07-26 15 43 15

File: doc/source/index.rst [hunk 1]
```diff
@@ -21,5 +21,5 @@ Key provided features:
    utilities
    design
    developer
-   api
```
Comment (kitchoi) [inline doc/source/index.rst]: accidentally removed 'api'?

Comment (stefanoborini) [inline doc/source/index.rst]: it's supposed to be (and already present) in developer

Comment (kitchoi) [inline doc/source/index.rst]: But I am used to seeing API at the top of the toc in other documentation, and 'developer' section is meant to contain more content than just a link to API.  Maybe it is just me :/

**Pull request merged**